csrc/ir_internal_nodes.h

```diff
@@ -1942,10 +1965,12 @@ class TORCH_CUDA_CU_API TensorDomain : public Val {
 
  private:
   const std::vector<IterDomain*> root_domain_;
+  const std::vector<IterDomain*> rfactor_domain_;
+  std::vector<IterDomain*> allocation_domain_;
```

Comment:
This is not used yet, right?

Reply:
Correct, it is just a placeholder for now.